redaxo/src/core/lib/form/form_base.php

```diff
@@ -0,0 +1,1257 @@
+<?php
+
+/**
+ * @package redaxo\core
+ */
+abstract class rex_form_base
+{
+    protected $name;
```

Comment:
in einer klasse, die für vererbung gemacht ist, sollten wir dann diese properties zumindest mit dem type-phpdoc versehen